assets/queries/common/passwords_and_secrets_in_infrastructure_code/data.json

```
@@ -0,0 +1,148 @@
+{
+  "ar": [
```

How about taking this opportunity to increase readability and rename this array to default passwords.
```suggestion
  "defaultPasswords": [
```